zebra-state/src/service/finalized_state/disk_format/upgrade/fix_tree_key_type.rs

```diff
@@ -46,6 +41,24 @@ pub fn run(
         .write_batch(batch)
         .expect("updating tree key formats should always succeed");
 
+    // These deletes can be slow due to tombstones for previously deleted keys,
```

Comment:
###### Optional Suggestion

```suggestion
    // The deletes below can be slow due to tombstones for previously deleted keys,
```